src/App.js

```diff
@@ -1,17 +1,18 @@
 import './App.css';
-// import { Component } from 'react';
+//import { Component } from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import Container from 'react-bootstrap/Container'
 import Header from './Components/Header.js';
 import Main from './Components/Main.js';
 import Footer from './Components/Footer.js';
 import HornedBeast from './data.json';
 
+
 function App() {
   return  (
     <Container>
       <Header title="Horned Animals"/>
-      <Main message="Make Your Beast Opinion Known" HornedBeast = {HornedBeast} />
+      <Main message="Make Your Beast Opinion Known" hornedBeast = {HornedBeast} />
       <Footer text="Your Opinion Counts!"/>
     </Container>
   );
```

Comment:
Great work so far, I know you are still workin' on this 👍 

Feedback:

- App should have a `selectedBeast` component.